src/gui/templates/contact.html

```diff
@@ -18,4 +18,33 @@ <h2>Kontakt</h2>
           <input type="submit" class="ig-button" value="Senden">
         </form>
       </div>
+
+      <h2>Team</h2>
+      {% is_staff %}
```

Comment:
Is `is_staff` defined anywhere? You probably need to iterate over all users and then check for the user attribute `is_staff`.

For example like this:
```
{% for user in author %}
{% if user.is_staff %}

render some HTML

{% endif %}
{% endfor %}
```